src/auth/jwt-auth.guard.ts

```diff
@@ -0,0 +1,4 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+@Injectable()
+export class JwtAuthGuard extends AuthGuard('jwt') {}
```

Comment:
PullProofAI

The code snippet provided for the `JwtAuthGuard` class looks correct and doesn't have any apparent issues.

```diff
No issues found.
```

This change seems to be a straightforward addition of a new file with a `JwtAuthGuard` class that extends `AuthGuard`. It follows the expected structure for creating a guard in NestJS. No issues were identified in this code snippet.


<!-- This is an auto-generated comment by PullProofAI -->